internal/pkg/comm/client_test.go

```diff
@@ -253,10 +253,9 @@ func TestNewConnection(t *testing.T) {
 			serverTLS: &tls.Config{
 				Certificates: []tls.Certificate{testCerts.serverCert},
 				ClientAuth:   tls.RequireAndVerifyClientCert,
-				MaxVersion:   tls.VersionTLS12, // https://github.com/golang/go/issues/33368
```

Comment:
Pretty sure this is no longer testing what it really means to test. IIUC, the grpc client needs to be coerced into reading from the server.